packages/api/server/ws.mts

```diff
@@ -788,11 +788,15 @@ async function getDefinitionLocation(payload: TsServerDefinitionLocationRequestP
     return null;
   }
 
-  wss.broadcast(`session:${session.id}`, 'tsserver:cell:definition_location:response', {
-    response: {
-      ...body[0],
-    },
-  });
+  const res = {
+    response: body[0]
+      ? {
+          ...body[0],
```

Comment:
Why do we need to create a copy of this object?

Reply:
needs to be typed as `{respinse: null}` or `{response: { ...body[0] }}` is there another way to do that which avoids a copy?

Reply:
Does `body[0]` work?

```typescript
const res = {
  response: body[0] ? body[0] : null, // or body[0] || null
};
```

Reply:
yep lmao this was something stupid I did when fighting zod very confused over something it was doing cleaned up